wiki/Beatmap/Difficulty/de.md

```diff
@@ -0,0 +1,82 @@
+
+## Schwierigkeitslevel
+
+*Siehe auch: [Benennung von Schwierigkeitsgraden](/wiki/Ranking_Criteria/Difficulty_naming)*
+
+Dies sind die typischen Level an Schwierigkeitsgraden, die eine Beatmap enthalten kann. Die spezifischen Definitionen und Anforderungen der einzelnen Level können in den entsprechenden [Ranking-Kriterien](/wiki/Ranking_Criteria) des jeweiligen Spielmodus gefunden werden.
+
+### ![](/wiki/shared/mode/osu.png) osu!
+
+- ![](/wiki/shared/diff/easy-o.png?20211215) Easy
+- ![](/wiki/shared/diff/normal-o.png?20211215) Normal
+- ![](/wiki/shared/diff/hard-o.png?20211215) Hard
+- ![](/wiki/shared/diff/insane-o.png?20211215) Insane
+- ![](/wiki/shared/diff/expert-o.png?20211215) Expert
+- ![](/wiki/shared/diff/expertplus-o.png?20211215) Expert+
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+
+- ![](/wiki/shared/diff/easy-t.png?20211215) Kantan
+- ![](/wiki/shared/diff/normal-t.png?20211215) Futsuu
+- ![](/wiki/shared/diff/hard-t.png?20211215) Muzukashii
+- ![](/wiki/shared/diff/insane-t.png?20211215) Oni
+- ![](/wiki/shared/diff/expert-t.png?20211215) Inner/Ura Oni
+- ![](/wiki/shared/diff/expertplus-t.png?20211215) Hell Oni
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+- ![](/wiki/shared/diff/easy-c.png?20211215) Cup
+- ![](/wiki/shared/diff/normal-c.png?20211215) Salad
+- ![](/wiki/shared/diff/hard-c.png?20211215) Platter
+- ![](/wiki/shared/diff/insane-c.png?20211215) Rain
+- ![](/wiki/shared/diff/expert-c.png?20211215) Overdose
+- ![](/wiki/shared/diff/expertplus-c.png?20211215) Overdose+
+
+### ![](/wiki/shared/mode/mania.png) osu!mania
+
+- ![](/wiki/shared/diff/easy-m.png?20211215) Easy
+- ![](/wiki/shared/diff/normal-m.png?20211215) Normal
+- ![](/wiki/shared/diff/hard-m.png?20211215) Hard
+- ![](/wiki/shared/diff/insane-m.png?20211215) Insane
+- ![](/wiki/shared/diff/expert-m.png?20211215) Expert
+- ![](/wiki/shared/diff/expertplus-m.png?20211215) Expert+
+
+## Schwierigkeitsstufen und Sternebewertung
+
+Seit dem 27. Juli 2021 ordnet die osu!-Webseite einer Schwierigkeitsstufe abhängig von ihrer Sternebewertung nach dem folgenden Spektrum automatisch eine Farbe zu:
+
+![Farbspektrum der Schwierigkeitsbewertung von osu!](/wiki/shared/star-rating/spectrum.png)
+
+In einigen Fällen wie beispielsweise [AIMod](/wiki/Client/Beatmap_editor/AiMod) werden Schwierigkeitsgrade ausgehend von der Sternebewertung in 6 Klassen wie folgt eingeteilt:
+
+- ![](/wiki/shared/diff/easy-o.png?20211215) Easy: 0.0★ bis 1.99★
```

Comment:
A minor nitpick I just noticed: star rating information could be written down with commas instead of full stops. osu! web does this too, and using commas in floats is more common for Europeans.

This also occurs in other articles (namely subarticles of `Medals`), so this change is out-of-scope here.